web/src/reducers/activities.js

```diff
@@ -399,8 +399,10 @@ const reducer = (state = initialState, action) => {
     case SELECT_APD: {
       const byKey = {};
       ((action.apd || {}).activities || []).forEach(a => {
-        byKey[a.key] = a;
-        byKey[a.key].meta = { expanded: a.name === 'Program Administration' };
+        byKey[a.key] = {
+          ...a,
+          meta: { expanded: a.name === 'Program Administration' }
```

Comment:
This started throwing an immutability error.  Kind of weird that it wasn't already doing that, in hindsight. 🤷‍♂️